lua/alex/lazy/theme.lua

```
@@ -0,0 +1,6 @@
+return { 
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)